scriptmodules/supplementary/skyscraper.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="skyscraper"
+rp_module_desc="Scraper for multiple frontends by Lars Muldjord"
+rp_module_help="IMPORTANT! In order for Skyscraper to work properly you need to exit your frontend. For EmulationStation you can do so by pressing F4.\nTo run Skyscraper in Simple Mode, exit your frontend and simply type 'Skyscraper' and press enter. Then answer the questions you are given and let Skyscraper do the work.\nIf you want better control check the available options with 'Skyscraper --help'.\n\nBe sure to check the elaborate documentation at github: https://github.com/muldjord/skyscraper"
+rp_module_licence="GPL3 https://raw.githubusercontent.com/muldjord/skyscraper/master/LICENSE"
+rp_module_section="opt"
+
+skysource="/home/$user/skysource"
+skyconfig="/home/$user/.skyscraper"
+
+function depends_skyscraper() {
+    local depends=(qt5-default)
+}
+
+function sources_skyscraper() {
+    if [[ -f "$skysource/Skyscraper" ]]; then
+        cd $skysource
+	./update_skyscraper.sh
+	sleep 5
+    else
+        mkdir $skysource
+        cd $skysource
+        curl https://raw.githubusercontent.com/muldjord/skyscraper/master/update_skyscraper.sh | bash
+    fi
```

Comment:
quotes - also please use wget instead of curl as we ensure wget is installed.

Reply:
I would prefer to not use a 3rd party script for the building but include the steps in our module.